src/screens/FeedScreen/Header.js

```diff
@@ -451,7 +593,22 @@ const styles = StyleSheet.create({
   avatarImage: {height: 48, width: 48, borderRadius: 24},
   postDetail: (isPostDetail) => ({
     paddingLeft: isPostDetail ? 10 : 0
-  })
+  }),
+  contentFeedName: {
+    flexDirection: 'row'
+  },
+  textFollow: {
+    color: COLORS.bluePrimary,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  },
+  textFollowing: {
+    color: COLORS.greySubtile1,
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles `textFollow` and `textFollowing` are almost identical, with only the color differing. Consider creating a base style for these text elements and then extending it with the specific color. This will make your code DRYer (Don't Repeat Yourself).
```diff
-  textFollow: {
-    color: COLORS.bluePrimary,
-    fontSize: normalizeFontSize(14),
-    fontStyle: 'normal',
-    fontWeight: '500'
-  },
-  textFollowing: {
-    color: COLORS.greySubtile1,
-    fontSize: normalizeFontSize(14),
-    fontStyle: 'normal',
-    fontWeight: '500'
-  }
+  textBase: {
+    fontSize: normalizeFontSize(14),
+    fontStyle: 'normal',
+    fontWeight: '500'
+  },
+  textFollow: {
+    ...textBase,
+    color: COLORS.bluePrimary,
+  },
+  textFollowing: {
+    ...textBase,
+    color: COLORS.greySubtile1,
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->